petisco/base/domain/errors/defaults/not_found.py

```diff
@@ -14,3 +15,27 @@ def get_specify_detail(self) -> str:
 class UserNotFound(DomainError):
     def get_specify_detail(self) -> str:
         return "User Not Found"
+
+
+class AggregateNotFoundError(DomainError):
+    def __init__(self, repository_name: str = None, entity_id: Uuid = None):
+        entity_id_str = f" ({entity_id.value})" if entity_id else ""
```

Comment:
If we're using now aggregate as main name, we should use `aggregate_id` instead of `entity_id`.

Reply:
Yes, well seen